scripts/base_network.py

```diff
@@ -298,9 +298,25 @@ def _set_electrical_parameters_links(links):
     links['p_min_pu'] = -p_max_pu
 
     links_p_nom = pd.read_csv(snakemake.input.links_p_nom)
-    links_p_nom["j"] = _find_closest_links(links, links_p_nom)
-
+    
+    #Filter links that are not in operation anymore    
+    links_p_nom['not_in_operation']=links_p_nom.Remarks.str.contains('Shut down', na=False) | links_p_nom.Remarks.str.contains('Replaced', na=False)
+    
+    links_p_nom = links_p_nom[links_p_nom.not_in_operation == False]
+ 
+    #add column for closest link id
+    links_p_nom["j"] = np.nan
+    
+    #find closest link for all links in links_p_nom
+    for index, row in links_p_nom.iterrows():        
+        links_p_nom.loc[[index],'j'] = _find_closest_links(links, links_p_nom.loc[[index]])
```

Comment:
So do I understand correctly that you get duplicated values in "j" if you applied 
```python 
    links_p_nom["j"] = _find_closest_links(links, links_p_nom)
```
?

Reply:
Hey, no duplicates. But you only get the first entry in the list (wiki export). By checking the list you see, that more entries belongs to one link.
Wiki list:
https://en.wikipedia.org/wiki/List_of_HVDC_projects#Europe

For example Gotland 1 and Gotland 2 belongs to the same link from the entso e gridkit. The function _find_closest_links only gives you one entry per link.

Reply:
Ah now I got you!